### PR TITLE
Add expansion support 

### PR DESCRIPTION
**Pull request by Shailin**
1. Added expansion support with splash screen.
2. Added user permission prompt. 
3. Added privacy policy.
4. Updated app icon.

### CODE REVIEW

File: platforms/android/app/src/main/java/com/maq/xprize/cci/hindi/SplashScreenActivity.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                DownloadExpansionFile.deleteDir(packageNameDir);
+            }
+            packageNameDir.mkdir();
+            _zip.unzip(unzipFilePath);
+            _zip.close();
+            toCallApplication();
+        } catch (IOException ie) {
+            System.out.println(ie);
+        }
+        return;
```
Comment (ankitk20) [inline platforms/android/app/src/main/java/com/maq/xprize/cci/hindi/SplashScreenActivity.java]: Remove _remove_

File: platforms/android/app/src/main/res/layout/activity_splash_screen.xml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    <TextView
+        android:id="@+id/one_time_extraction_text"
+        android:layout_width="wrap_content"
+        android:layout_height="60dp"
+        android:layout_marginBottom="24dp"
+        android:gravity="center_vertical"
+        android:text="@string/content_extraction"
+        android:textColor="@android:color/black"
+        android:textSize="30sp"
```
Comment (ankitk20) [inline platforms/android/app/src/main/res/layout/activity_splash_screen.xml]: Reduce the size. Take a look at Chimple repo

File: platforms/android/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -0,0 +1,92 @@
+<?xml version='1.0' encoding='utf-8'?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.maq.xprize.cci.hindi"
+    android:hardwareAccelerated="true"
+    android:versionCode="10000"
```
Comment (ankitk20) [inline platforms/android/app/src/main/AndroidManifest.xml]: Make version code to 1

File: platforms/android/app/src/main/java/com/maq/xprize/cci/hindi/SplashScreenActivity.java [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+        setContentView(R.layout.activity_splash_screen);
+        if (ContextCompat.checkSelfPermission(this, Manifest.permission.WRITE_EXTERNAL_STORAGE)
+                != PackageManager.PERMISSION_GRANTED ||
+                ContextCompat.checkSelfPermission(this, Manifest.permission.ACCESS_FINE_LOCATION)
+                        != PackageManager.PERMISSION_GRANTED) {
+            // Permission is not granted
+            ActivityCompat.requestPermissions(this, new String[]{Manifest.permission.WRITE_EXTERNAL_STORAGE,
+                    Manifest.permission.ACCESS_FINE_LOCATION}, 1);
+        } else {
+            new DownloadFile().execute(null, null, null);
```
Comment (akashdabhi03) [inline platforms/android/app/src/main/java/com/maq/xprize/cci/hindi/SplashScreenActivity.java]: Add the logic to check whether the expansion file needs to be extracted even if the success.txt is present.
You can refer to onebillion's PR

File: package-lock.json [hunk 5]
```diff
@@ -0,0 +1,583 @@
+{
+    "name": "com.example.cciapp2",
```
[the remark this answers is not in the document]
Comment (Shailin) [reply package-lock.json]: Updated

File: platforms/android/.gitignore [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+# Comment next line if keeping position of elements in Navigation Editor is relevant for you
+.idea/navEditor.xml
+
+# Gradle files
+.gradle/
+build/
+
+# Signing properties file
+release-signing.properties
```
[the remark this answers is not in the document]
Comment (Shailin) [reply platforms/android/.gitignore]: I have added in the parent folder .gitignore. Added in the Android platform .gitignore

File: platforms/android/app/src/main/java/com/maq/xprize/cci/hindi/SplashScreenActivity.java (hunk 4 above)
Comment (Shailin) [reply platforms/android/app/src/main/java/com/maq/xprize/cci/hindi/SplashScreenActivity.java]: Will require update and testing after adding the logic. Kept in the backlog for now

**Pull request merged**